src/profile/components/editProfileForm/EditProfileForm.tsx

```diff
@@ -211,11 +218,58 @@ function EditProfileForm(props: Props) {
                 options={countryOptions}
                 labelText={t('profileForm.country')}
               />
-              <br />
             </div>
+            <div className={styles.linebreak} />
+            <h2 className={styles.additionalInfo}>
+              {t('profileForm.additionalInfo')}
+            </h2>
+            <FieldArray
+              name="emails"
+              render={(arrayHelpers: ArrayHelpers) => (
+                <React.Fragment>
+                  <div className={styles.formFields}>
+                    {formikProps?.values?.emails.map(
+                      (email: Email | PrimaryEmail, index: number) => (
+                        <div key={index} className={styles.formField}>
+                          <Field
+                            as={TextInput}
+                            name={`emails.${index}.email`}
+                            id={`emails.${index}.email`}
+                            labelText={t('profileForm.email')}
+                          />
+                          {!email.id && (
+                            <button
```

Comment:
Looking at the code, I get the sense that this button may be styled in a weird way on some platforms, for instance mobile. As far as I can tell, you are using the default button element with one class that just adjusts margin and colour 🤔If buttons are reset in a normalize.css etc, then I can see how it could work.

Reply:
It seems to working on mobile resolution, nothing is exploding.

Reply:
I think you'd have to use a mobile browser. For instance iOS Safari tends to force some sort of background and borders 🤔

Reply:
I'll give that a go before merging.

Reply:
But maybe if there's nothing on Chrome the chance is quite small.